Add December Spotlight (#24)
mdn/mdn-contributor-spotlight: contributors/yash-raj-bharti/index.md

---
contributor_name: Yash Raj Bharti
folder_name: yash-raj-bharti
is_featured: true
img_alt: A person standing next to a colorful graffiti-painted Android statue, with the Google headquarters in the background. The person is wearing a lanyard on a sunny day.
usernames:
  github: yashrajbharti
quote: MDN resonates with my passion to build a consistent and open web, where developers can learn and grow.
---

> Hi! I'm' Yash, a UXE and a Senior Mentor of the Liquid Galaxy, where I work on GIS applications, making immersive data viz projects on Google Maps, Streetview, and Earth. Liquid Galaxy is a Google-founded open-source project and was my primary reason for falling in love with open-source. I work on the web: HTML, CSS, and JS, and I believe that honing strong basics can make even the most complex tasks easy.

Reviewer: ```suggestion
Hi! I'm Yash, a UXE and a Senior Mentor of the Liquid Galaxy, where I work on GIS applications, making immersive data viz projects on Google Maps, Street View, and Earth. Liquid Galaxy is a Google-founded open-source project and was my primary reason for falling in love with open-source. I work on the web with HTML, CSS, and JS, and I believe that honing strong basics can make even the most complex tasks easy.
```

> I still remember my first contribution to MDN. I was so proud of myself. Even though the fix was small, the cheer from Will Bamberg(a contributor) at the time meant a lot. It was for a [PWA](https://github.com/mdn/content/pull/29488), and I said: iTunes value associated with the App Store for platform members of related apps has been dropped since it's misleading and currently not supported by Safari. The list has been updated as per the W3C wiki.

Reviewer: ```suggestion
I still remember my first contribution to MDN. I was so proud of myself. Even though the fix was small, the cheer from Will Bamberg at the time meant a lot. It was for [PWA docs](https://github.com/mdn/content/pull/29488), where I corrected that the iTunes value associated with the App Store for platform members of related apps has been dropped since it's currently not supported by Safari.
```

> Since then, I have been lurking to find a way to contribute to MDN, I also contributed to web.dev to learn more. I am not new to MDN W3C, or Web.dev (HTML5Rocks), and have been learning like a nerd for the last 9 years, I was a couple of years back I started contributing, felt like someone helping the space, making explainers, worked alongside people like Adam Argyle from [CSS motion blur](https://github.com/w3c/csswg-drafts/issues/11134) and Thomas Steiner(Project Fugu). I discovered, that when I put my passion and heart into it, I can achieve, the grit to solve challenges isn't new to me.

Reviewer: ```suggestion
Since then, I have been lurking to find a way to contribute to MDN. I also contributed to [web.dev](https://web.dev/) to learn more. I am not new to MDN, W3C, or web.dev (HTML5Rocks), and have been learning like a nerd for the last 9 years, but a couple of years back, I started finally contributing. I felt like helping the community, making explainers, and I worked alongside people like Adam Argyle on a [CSS motion blur proposal](https://github.com/w3c/csswg-drafts/issues/11134) and Thomas Steiner ([Project Fugu](https://www.chromium.org/teams/web-capabilities-fugu/)). I discovered, that when I put my passion and heart into something, I can achieve it. The grit to solve challenges isn't new to me.
```

> My work at Liquid Galaxy, documentation and creating an innovative UX to control Google Earth using Google Maps (a first ever!) led me to get invited as a Speaker to Google, CA, USA (my first time outside India), where I got to give a Talks at Google, and most importantly network! I met Kristi Progri who was previously a Mozilla Representative, and I felt that's so cool, I read from Mozilla every day and was confident too that I could do something.

Reviewer: ```suggestion
My work at Liquid Galaxy, documentation, and creating an innovative UX to control Google Earth using Google Maps led me to get invited as a Speaker to Google, CA, USA, which was my first time outside India. I got to speak at Talks at Google, and most importantly, to network! I met Kristi Progri who was previously a Mozilla Representative, and I thought "that's so cool, I read MDN every day" and I was confident that I could get involved, too.
```

> This is when after an incredible journey to Google, knowing the roots of the liquid galaxy from 2008, I was mesmerized and was full of energy to do something, thus I went to MDN and saw some simpler bugs like buttontext, where I met Estelle Weyl (and so thankful I did, as I had all the passion and pure knowledge waiting to test the waters). They helped me with the contributions by stating how buttontext for accessibility reasons will be fixed by filing it to Webkit. After some PRs like this, I mailed them asking to create an issue about the `<model>` tag, what result? They told me it's still a WIP but there are some Widely Baseline Available PRs I can contribute to. They "trusted me", created a Google Sheet, and gave me edit access. And I delivered. The rest is history, I found my chance and took it, I knew Estelle because I had read her book on CSS, and was amazed to work alongside her on this. [This sheet](https://docs.google.com/spreadsheets/d/1O9tOVsZDF26A8Rag38rAhGWRH5l6VTbDCJIknyKzO_0/edit?gid=276647839#gid=276647839) holds value to me and is the most updated thing on the work we are making.

Reviewer: ```suggestion
After an incredible journey to Google, knowing the roots of the Liquid Galaxy from 2008, I was mesmerized and was full of energy to do something, so I went to MDN and saw some small bugs like `buttontext` details, where I met Estelle Weyl (and I'm so thankful I did, as I had all the passion and pure knowledge waiting to test the waters). They helped me with the contributions by stating how `buttontext` for accessibility reasons will be fixed by filing a bug in WebKit. After some PRs like this, I mailed them asking to create an issue about the `<model>` tag. They told me it's still a WIP but there are some Baseline 'Widely Available' PRs I can contribute to. They trusted me, created a Google Sheet, and gave me edit access. And I delivered. The rest is history, I found my chance and took it, I knew Estelle because I had read her book on CSS, and was amazed to work alongside her on this. [This sheet](https://docs.google.com/spreadsheets/d/1O9tOVsZDF26A8Rag38rAhGWRH5l6VTbDCJIknyKzO_0/edit?gid=276647839#gid=276647839) holds value to me and is the most up-to-date record of the work we are doing.
```

> I love it because just think about how many future developers can be helped with it, those who truly learn like me and do not rely on crash courses to just get it "done", I thank Estelle for my journey and I want to stay in this community and help make mdn the library, the well-organized library I know It as, where you just know where each book is.

Reviewer: ```suggestion
I love it because just think about how many future developers can be helped with it, those who truly learn like me and do not rely on crash courses to just get it "done". I thank Estelle for my journey and I want to stay in this community and help make MDN the library, a well-organized library I know it as, where you just know where each book is.
```

## How did you start using MDN?

> I started just like others, but using it as a source of learn, from those little popups that say "MDN reference" in VScode, all the way to contributing and making the docs right, wherever I can find a way to contribute, to hopefully be a part of it.

Reviewer: ```suggestion
I started just like others, but using it as a source of learning, from those little tooltips that say "MDN reference" in VS Code, all the way to contributing and making the docs correct, wherever I can find a way to contribute, to hopefully be a part of it.
```

## What do you like about MDN?

> I like it has all the implementation knowledge needed, when documenting what I truly got to know from will was, we are not just paraphrasing w3c specs, we are building it with an abstraction keeping in mind developers are our end users. (and UXEs like me too!)

Reviewer: ```suggestion
I like that it has all the implementation knowledge needed. When documenting, what I truly got to know from Will was, we are not just paraphrasing W3C specs, we are building it with an abstraction keeping in mind developers (and UXEs like me too) are our end users.
```

## Why do you contribute to Open Source or MDN?

> Anything open source is where we should say "why should we not contribute" There's all the reasons in the world to help and let everyone prosper, and that's what I love, whether is for Google or Mozilla.

Reviewer: ```suggestion
Anything open source is where we can ask "why should we not contribute"? There's all the reasons in the world to help and let everyone prosper, and that's what I love, whether it's for Google or Mozilla.
```

## What do you enjoy about contributing to MDN?

> I loved making DOMQuad as I was learning first hand and I believe I will become an encyclopaedia as a by-product of helping others! That's rad.

Reviewer: ```suggestion
I loved making `DOMQuad` as I was learning first hand and I believe I will become an encyclopedia as a by-product of helping others! That's rad.
```

> I loved making `DOMQuad` as I was learning first hand and I believe I will become an encyclopedia as a by-product of helping others! That's rad.

Reviewer: ```suggestion
I loved making `DOMQuad` as I was learning first hand and I believe I will become an encyclopedia as a by-product of helping others! That's rad.

```
